core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/StoreCallbacksImpl.scala

```diff
@@ -174,6 +180,94 @@ object StoreCallbacksImpl extends StoreCallbacks with Logging with Serializable
     SnappyStoreHiveCatalog.registerRelationDestroy()
   }
 
+  override def performConnectorOp(ctx: Object): Unit = {
+
+    val context = ctx.asInstanceOf[LeadNodeSmartConnectorOpContext]
+
+    context.getType match {
+      case LeadNodeSmartConnectorOpContext.OpType.CREATE_TABLE =>
```

Comment:
Can we refactor these cases to methods. If we add further operations like (alter table etc ) this method will become big.